sycl/unittests/Extensions/CommandGraph.cpp

```diff
@@ -694,3 +760,175 @@ TEST_F(CommandGraphTest, MakeEdgeErrors) {
   // Re-check graph structure to make sure the graph state has not been modified
   CheckGraphStructure();
 }
+
+TEST_F(CommandGraphTest, MultiThreadsBeginEndRecording) {
```

Comment:
Rather than prefixing all these tests with `MultiThreads`, we can define a new gtest fixture class that derives from `CommandGraphTest` with some common functionality then create the tests based on that:

e.g

```cpp
class MultiThreadGraphTest : public CommandGraphTest {

 MultiThreadGraphTest() : CommandGraphTest(), NumThreads(std::thread::hardware_concurrency()), SyncPoint(NumThreads) {
   Threads.reserve(NumThreads);
}


unsigned NumThreads; 
Barrier SyncPoint;
std::vector<std::thread> Threads;
};

TEST_F(MultiThreadGraphTest, BeginEndRecording) {
```

Reply:
Nice suggestion. Thanks.